src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -194,4 +251,27 @@ class KeyPhraseExtractionSuiteV4 extends TestBase with DataFrameEquality with Te
         row.toSeq.foreach { col => println(col) }
       }
     }
+
+  lazy val df3: DataFrame = Seq(
+    ("en","Hello world. This is some input text that I love."),
+    ("es", "La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+    ("fr", "Bonjour tout le monde")
+  ).toDF("lang","text" )
+
+  test("Keyphrase - batch usage"){
+    lazy val extractor2: TextAnalyticsKeyphraseExtraction = new TextAnalyticsKeyphraseExtraction(options)
+      .setSubscriptionKey(textKey)
+      .setEndpoint("https://eastus.api.cognitive.microsoft.com/")
+      .setInputCol("text")
+      .setLangCol("lang")
+      .setBatchSize(8)

```

Comment:
if the input contains only 3 documents and the batch size is set as 8,  there is no difference comparing with the default setting of 5? So the result will look the same.